frontend/components/nav_bar/nav_bar.module.scss

```diff
@@ -0,0 +1,46 @@
+@use "~@components/design_system/design_system.module.scss" as ds;
+
+.nav_bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  background-color: ds.$color-black;
+}
+
+.image {
```

Comment:
Should be using multiples of ds.$measurement-baseline-grid rather than px directly. The use of a baseline grid is part of our design system

Reply:
Feel free to change up the measurements slightly to conform to multiples of the baseline grid (8px) as the Figma file is in a draft state at the moment. Multiples of 0.5 (4px) are also fine.

Reply:
Ah, okay. I can fix that